JotunnLib/Utils/SimpleJVL.cs

```diff
@@ -0,0 +1,272 @@
+        /// <param name="prefabNew">Case sensitive name of the new .prefab.</param>
+        /// <param name="prefabOld">Case sensitive name of the .prefab you are cloning.</param>
+        /// <param name="name">Token name of the new item.</param>
+        /// <param name="description">Description of the item, shows up in tooltip.</param>
+        public void AddClonedItem(string prefabNew, string prefabOld, string name, string description)
+        {
+            CustomItem Item = new CustomItem(prefabNew, prefabOld);
+            ItemDrop ItemDrop = Item.ItemDrop.m_itemData.m_shared;
+            ItemDrop.m_name = name;
+            ItemDrop.m_description = description;
+            ItemManager.Instance.AddItem(Item);
+        }
+
+        /// <summary>
+        ///     Adds a <see cref="Piece"/> to ObjectDB with inherited properties by default, and builds a new recipe for it.
+        /// </summary>
+        /// <param name="prefabName">Case sensitive name of the new <see cref="Piece"/>.</param>
+        /// <param name="name">Token name of the new item.</param>
+        /// <param name="description">Description of the item, shows up in tooltip.</param>
+        /// <param name="pieceTable">Case sensitive name of <see cref="PieceTable"/> assigned to the recipe.</param>
+        /// <param name="craftingStation">Case sensitive name of the <see cref="CraftingStation"/> required </param>
+        /// <param name="inputs">Recipe config as a <see cref="RequirementConfig"/></param>
+        public void AddPiece(string prefabName, string name, string description, GameObject prefab,
+            string pieceTable = "piece_HammerPieceTable", string craftingStation = "",
+            bool isAllowedInDungeons = false, params RequirementConfig[] inputs)
+        {
+            AddStationPiece(prefabName, name, description);
+            AddPieceRequirements(prefab, pieceTable, craftingStation, isAllowedInDungeons, inputs);
+        }
+
+        /// <summary>
+        ///     Adds a <see cref="CraftingStation"/> to ObjectDB with inherited properties by default.
+        /// </summary>
+        /// <param name="prefabName">Case sensitive name of the new <see cref="Piece"/>.</param>
+        /// <param name="name">Token name of the new item.</param>
+        /// <param name="description">Description of the item, shows up in tooltip.</param>
+        public void AddStationPiece(string prefabName, string name, string description)
+        {
+            GameObject prefabObject = AddItem(prefabName, name, description);
+            CraftingStation reflect = prefabObject.GetComponent<CraftingStation>();
+            if (reflect == null)
+            {
+                GameObject er = GameObject.Find("piece_workbench");
+                CraftingStation craft = er.GetComponent<CraftingStation>();
+                prefabObject.AddComponent<CraftingStation>();
+                CraftingStation Craft = prefabObject.GetComponent<CraftingStation>();
```

Comment:
Why not just clone the CraftingStation component instead of copying field by field? Hard to maintain...